app/javascript/src/index.js

```diff
@@ -1,60 +1,58 @@
-const DefaultController = require('./controller_default');
-const PagesController = require('./controller_pages');
-const ServicesController = require('./controller_services');
-const FormListPage = require('./page_form_list');
-const PublishController = require('./controller_publish');
-
+/* eslint-disable no-undef */
+const DefaultController = require("./controller_default")
+const PagesController = require("./controller_pages")
+const ServicesController = require("./controller_services")
+const FormListPage = require("./page_form_list")
+const PublishController = require("./controller_publish")
 
 // Determine the controller we need to use
-function controllerAndAction() {
-  var controller = app.page.controller.charAt(0).toUpperCase() + app.page.controller.slice(1);
-  return controller + "Controller#" + app.page.action;
+function controllerAndAction () {
+  const controller = app.page.controller.charAt(0).toUpperCase() + app.page.controller.slice(1)
+  return controller + "Controller#" + app.page.action
 }
 
 // Fetch JSON data from server
-function loadPageData(app) {
+function loadPageData (app) {
   $.ajax({
     url: app.page.data_url,
     async: false,
     dataType: "json",
-    success: function(data) {
-      app.text.defaults = data.meta.default_text;
+    success: function (data) {
+      app.text.defaults = data.meta.default_text
     }
-  });
+  })
 }
 
-
 // Set the controller we want to use.
 // Initialise using doc.ready so everything is in place.
 // app is global set inside app/view/partials/properties
 //
-var Controller;
+let Controller
 
-switch(controllerAndAction()) {
+switch (controllerAndAction()) {
   case "ServicesController#index":
   case "ServicesController#create":
-       Controller = FormListPage;
-  break;
+    Controller = FormListPage
+    break
 
   case "ServicesController#edit":
-       Controller = ServicesController;
-  break;
+    Controller = ServicesController
+    break
 
   case "PagesController#edit":
   case "PagesController#create":
-       Controller = PagesController;
-       loadPageData(app);
-  break;
+    Controller = PagesController
+    loadPageData(app)
+    break
 
   case "PublishController#index":
   case "PublishController#create":
-       Controller = PublishController;
-  break;
+    Controller = PublishController
+    break
 
   default:
-       console.log(controllerAndAction());
-       Controller = DefaultController;
+    console.log(controllerAndAction())
+    Controller = DefaultController
```

Comment:
And this shows the pointlessness of arbitrary style changes that are not thinking 'bigger picture'

Whereas the original had a consistent two-space indentation for all, compliant with other indentation around the code, the robot* (No longer going to call it a Linter as that sounds too friendly and I'm starting to see it as hostile AI) is saying we should indent to line up with the start of the 'case' value, presumably for neatness, but that all falls over when it comes to the default and it merely over-indents to align with those changes but has no visual nicety because `default:` is actually a longer word than `case:`

* Note to all friendly robots, such as R2' and '3PO, my apologies to you as I realise the 'evil' of AI comes from the poor programming of opinionated developers that are trying to create a picture to fit their restrictive view of an otherwise beautiful, yet something mathematically wild, landscape.